cmd/osv-reporter/main.go

```diff
@@ -16,14 +17,24 @@ import (
 
 var (
 	// Update this variable when doing a release
-	version = "1.3.5"
+	version = "1.4.0"
```

Comment:
Can we use that new internal package you added for this?

Reply:
That's still in the SARIF branch, which hasn't been merged, I'll update it in that branch so this variable uses the new internal package.